controller/environmentApi.go

```diff
@@ -48,7 +48,6 @@ func (api *API) EnvironmentApiCreate(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		api.respondWithError(w, err)
 	}
-	defer r.Body.Close()
```

Comment:
According to the document (https://golang.org/pkg/net/http/#Request) 

```
// Body is the request's body.
//
// For client requests a nil body means the request has no
// body, such as a GET request. The HTTP Client's Transport
// is responsible for calling the Close method.
//
// For server requests the Request Body is always non-nil
// but will return EOF immediately when no body is present.
// The Server will close the request body. The ServeHTTP
// Handler does not need to.
Body io.ReadCloser
```

server http handler does not need to close request body, so I removed this line for consistency.